tools/apps/update_homebrew/lib/src/impl.dart

```diff
@@ -65,10 +71,12 @@ Future<Map<String, String>> _getCurrentRevisions(String repository) async {
 Future<Map<String, Map>> _getHashes(Map<String, String> revisions) async {
   var hashes = <String, Map>{};
   for (var channel in supportedChannels) {
-    hashes[channel] = {};
-    for (var file in _files[channel]) {
-      var hash = await _getHash256(channel, revisions[channel], file);
-      hashes[channel][file] = hash;
+    hashes[channel] = {'mac': {}, 'linux': {}};
```

Comment:
Please remove `'mac': {}, 'linux': {}`, that doesn't seem to be used anywhere.

Reply:
Both are used in code block starting line 93.

Reply:
Sorry, but I fail to see how they are used.

For instance:
```
hashes['stable'][_x64Files['linux']]
_x64Files['linux'] -> 'sdk/dartsdk-linux-x64-release.zip'
```
So this is:
`hashes['stable']['sdk/dartsdk-linux-x64-release.zip'] -> <hash>`

`hashes` is a map from channel (`'dev'` or `'stable'`) to a map from filename to the hash for the file, because it's values are assigned like this:
```
hashes[channel][file] = hash; // not hashes[channel]['mac'][file] = hash;

// hashes['dev'] will contain:
{
  'mac': {},
  'linux': {},
  'sdk/dartsdk-linux-x64-release.zip': <hash>,
  'sdk/dartsdk-linux-ia32-release.zip': <hash>,
  'sdk/dartsdk-macos-x64-release.zip': <hash>,
  'sdk/dartsdk-macos-ia32-release.zip': <hash>,
};
```
So, I don't see what the empty `'mac': {}, 'linux': {}` maps add (they would be needed if the filenames were the same on the different OSs).

Reply:
Useless initialisation removed

Reply:
Thanks!